### PR TITLE
Fix importing entities from URL

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
Test Plan

Test importing entities from URL:
* Edit > Import Entities from URL and enter https://hifi-content.s3.amazonaws.com/patrickmanalich/mirrorFolder/m.json
* The entities editor should open if it isn't already open.
* Content should be rezzed in front of you.
* Check program log for JavaScript errors.

Test exporting and importing entities to and from disk.
* Use Edit > Export Entities and Edit > Import Entities to export and export one or more entities.
* The entities editor should open if it isn't already open when you import.
* Verify that the entities exported to JSON are rezzed in front of you when imported from JSON.
* Check program log for JavaScript errors.

**Comment by ctrlaltdavid** [comment]
QA ready

File: scripts/system/edit.js [hunk 1]
```diff
@@ -1476,10 +1476,19 @@ function onFileSaveChanged(filename) {
 
 function onFileOpenChanged(filename) {
     // disconnect the event, otherwise the requests will stack up
-    Window.openFileChanged.disconnect(onFileOpenChanged);
+    try {
+        // Not all calls to onFileOpenChanged() connect an event.
+        Window.openFileChanged.disconnect(onFileOpenChanged)
```
Comment (misslivirose) [inline scripts/system/edit.js]: I believe eslint wants a semi-colon here, otherwise  changes look good

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4247/HighFidelity-Beta-PR11622-f74c33f597d85f80c9aedbde5f50d281a69d649b.exe

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4248/HighFidelity-Beta-PR11622-f74c33f597d85f80c9aedbde5f50d281a69d649b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4248/HighFidelity-Beta-PR11622-f74c33f597d85f80c9aedbde5f50d281a69d649b.exe

File: scripts/system/edit.js (hunk 1 above)
Comment (ctrlaltdavid) [reply scripts/system/edit.js]: Done, thanks.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4249/HighFidelity-Beta-PR11622-5b6d33bcb23fc530006e5af7af04dcd4a7545254.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4249/HighFidelity-Beta-PR11622-5b6d33bcb23fc530006e5af7af04dcd4a7545254.exe

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Passed
## Test importing entities from URL:
Test | Result
--- | ---
Edit > Import Entities from URL and enter https://hifi-content.s3.amazonaws.com/patrickmanalich/mirrorFolder/m.json | Pass
The entities editor should open if it isn't already open. | Pass
Content should be rezzed in front of you. | Pass
Check program log for JavaScript errors. | Pass

## Test exporting and importing entities to and from disk.
Test | Result
--- | ---
Use Edit > Export Entities and Edit > Import Entities to export and export one or more entities. | Pass
The entities editor should open if it isn't already open when you import.| Pass
Verify that the entities exported to JSON are rezzed in front of you when imported from JSON. | Pass
Check program log for JavaScript errors. | Pass 

All good.

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Pull request merged**